stage2/BuildDependency.scala

```diff
@@ -27,11 +27,13 @@ case class BuildDependency(context: Context) extends TriggerLoop{
   def triggerLoopFiles = root.triggerLoopFiles
   override final val needsUpdate = build.needsUpdate
   def targetClasspath = ClassPath(Seq())
+  private[cbt] def serialize = context.projectDirectory.toString 
```

Comment:
I don't think serialize is a good name for this method. It does not serialize BuildDependency, but only one part of it. let's just use projectDirectory above